iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -62,8 +62,25 @@ PortManager::PortManager(RouDiMemoryInterface* roudiMemoryInterface) noexcept
     }
     auto introspectionMemoryManager = maybeIntrospectionMemoryManager.value();
 
+    popo::PublisherOptions registryPortOptions;
+    registryPortOptions.historyCapacity = 1U;
```

Comment:
Please defined `static constexpr ` member variables and use them here.

Reply:
I agree with @MatthiasKillat. IMHO it would be too much overhead to define these local values always `static constexpr` instead we should really try to make all the building blocks in iceoryx `constexpr`'able e.g. `cxx::string` or the `popo::PublisherOptions` would be good candidates to start with.

Reply:
@mossmaurice Lets state it differently. We have to follow coding guidelines which explicitly state: no magic numbers.

We have three options now.
1. Let the magic number stick here and add a exception as comment for the static code analysis (which makes the code less readable and clutters it with useless comments).
2. Create a `static constexpr ` member variable and use it here. Then we do not have to write an exception comment for the static code analysis and the code is more readable.
3. Do nothing and create technical debt which someone has to fix - so we create more work for the future. 

The choice is yours...

Reply:
@elfenpiff I don't really see an issue here, also in terms of maintainability. Let's ping @elBoberido to get his opinion

Reply:
As I stated, the choice is yours.